Makefile

```diff
@@ -114,6 +114,7 @@ test-nats-e2e:
 test-sdks-e2e:
 test-reduce-e2e:
 test-api-e2e:
+test-udsource-e2e:
```

Comment:
This is new to me. Thanks for updating. Could you also update the list? Some of them are no longer useful. e.g. 1. e2e-suite-1 and 2 were renamed. 2. we also added transformer-e2e and diamond-e2e etc.